assets/js/plugin-check-admin.js

```diff
@@ -108,6 +108,13 @@
 		for ( let i = 0; i < categoriesList.length; i++ ) {
 			categoriesList[ i ].disabled = false;
 		}
+
+		//Adding Open All/Collapse All button
```

Comment:
```suggestion
		// Adding Open All/Collapse All button.
```

Inline comment should be end with period.

🔢 Apply same to other places also.